emit cell attribute into kore definition

dwightguth opened the pull request:
This PR adds the `cell` attribute back to the KORE definition in order to allow downstream usage in Pi Squared.

Commit:
emit cell attribute into kore definition

Baltoli commented:
Looks like this affects some Pyk tests that reference the attribute subsystem; let me know if you want a hand from @tothtamas28 or me to sort those out.

tothtamas28 commented:
> Looks like this affects some Pyk tests that reference the attribute subsystem; let me know if you want a hand from @tothtamas28 or me to sort those out.

Here I think: https://github.com/runtimeverification/k/blob/ff056270cc62c50284c1865f5744b499f8d9a36b/pyk/src/pyk/konvert/_module_to_kore.py#L726

Commit:
attempt pyk test fix

Commit:
delete flaky test

This test seems to fail intermittently due to some kind of
nondeterminism in the haskell backend. It has failed on this PR for
spurious reasons, and has failed similarly in the past. I believe it
ought to be deleted since its incremental value individually is tiny

tothtamas28 commented:
69fcd40: this seems to be another instance of runtimeverification/haskell-backend#3761.

rv-jenkins committed:
Merge branch 'develop' into attribute
